src/ir_Hitachi.h

```diff
@@ -358,15 +384,17 @@ class IRHitachiAc3 {
 #ifndef UNIT_TEST
 
  private:
-  IRsend _irsend;
-#else
-  IRsendTest _irsend;
-#endif
-  // The state of the IR remote in IR code form.
-  uint8_t remote_state[kHitachiAc3StateLength];
+  IRsend _irsend;  ///< Instance of the IR send class
+#else  // UNIT_TEST
+  /// @cond IGNORE
+  IRsendTest _irsend;  ///< Instance of the testing IR send class
+  /// @endcond
+#endif  // UNIT_TEST
```

Comment:
Maybe this kind of update could be done to all classes at once (more or less search and replace)

Reply:
I have thought of stuffing all the similar bits into a base class and just inheriting it.
I need to find out if that causes any memory and/or flash overhead.

That said, honestly that bit of repetition is the least of my drudgery.